artifactor/plugins/ostriz.py

```diff
@@ -30,7 +30,7 @@ def configure(self):
 
     @ArtifactorBasePlugin.check_configured
     def ostriz_send(
-            self, artifacts, test_location, test_name, slaveid, polarion_ids, run_id,
```

Comment:
**QUESTION** why limit it to one instead of all?

Reply:
I guess it is to ensure 1:1 mapping? (...*wildguess*)

Reply:
@mfalesni @RonnyPfannschmidt That is correct. I believe a case should not have more than 1 ID. The idea here would be to use the polarion_id as id for cm-qe to import their tests into polarion. I looked through the repo and I do not see anyone else using the polarion marker so I do not believe this would affect any other team.